### PR TITLE
Simpler language in footer

### PR DESCRIPTION
**Pull request by jrubenoff**
screen shot 2016-01-04 at 2 09 45 pm

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 en:
   dvl_core:
     footer:
-      about: " is an application by <a href='https://www.dobt.co' target='_blank'>The Department of Better Technology</a>."
-      about_no_app: "Built by <a href='https://www.dobt.co' target='_blank'>The Department of Better Technology</a>."
-      status: Service Status
+      about: " is made by <a href='http://www.dobt.co' target='_blank'>The Department of Better Technology</a>."
```
Comment (ajb) [inline config/locales/en.yml]: Looks like you didn't properly merge my changes -- these links should be `https`

**Comment by ajb** [comment]
"made by" sounds kinda awkward, like we're trying to be :dizzy: artisanal :dizzy: or something.

Maybe "built by"?

File: config/locales/en.yml (hunk 1 above)
Comment (jrubenoff) [inline config/locales/en.yml]: Sorry, copied this over from an old branch. Fixed

**Comment by jrubenoff** [comment]
Sounds good, changed it.

**Comment by ajb** [comment]
:+1: that was easy!

**Pull request merged**